src/read/pdb/utils.rs

```diff
@@ -0,0 +1,40 @@
+pub(crate) fn fast_parse_u64_from_string(s: &str) -> u64 {
+    let bytes = s.as_bytes();
+    let mut result: u64 = 0;
+    let mut i = 0;
+    let len = bytes.len();
+
+    // Skip leading whitespace
+    while i < len && bytes[i].is_ascii_whitespace() {
+        i += 1;
+    }
+
+    // Parse digits
+    while i < len && bytes[i].is_ascii_digit() {
```

Comment:
Could this be rewritten as an iterator? `bytes.skip_while(|b| b.is_ascii_whitespace()).take_while(|b| b.is_ascii_digit())`. Only if you think the code will improve from that change.